test/unit/gen/certificate.go

```diff
@@ -261,3 +261,9 @@ func SetCertificateRevisionHistoryLimit(limit int32) CertificateModifier {
 		crt.Spec.RevisionHistoryLimit = &limit
 	}
 }
+
+func SetCertificateAdditionalOutputFormats(additionalOutputFormats []v1.AdditionalOutputFormat) CertificateModifier {
+	return func(crt *v1.Certificate) {
+		crt.Spec.AdditionalOutputFormats = additionalOutputFormats
+	}
+}
```

Comment:
This is a good candidate for a variadic function like used elsewhere in these generators.

```suggestion
func SetCertificateAdditionalOutputFormats(additionalOutputFormats ...v1.AdditionalOutputFormat) CertificateModifier {
	return func(crt *v1.Certificate) {
		crt.Spec.AdditionalOutputFormats = additionalOutputFormats
	}
}
```